clients/cloud/python/consumer.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I'm not sure about this change.  On my system, where I have a python 2.7 installation, `env python` resolves to python 2, and cannot locate the confluent-kafka package or run the examples.  I'm not sure of the recommended method for making this work on a system like mine, or if it should be a system specific configuration change instead of this.  @rnpridgeon can you help me if this is a valid change or should users be doing something local that I can document?

Reply:
I defer to @rnpridgeon but I believe users  often alias `python` to their local `python2` or `python3`, so consider leaving as `python`.

Reply:
IIUC `env` does not reference aliases

Reply:
In my case I used pip3 (aliased on my machine from pip to pip3) to install confluent-kafka, but python 2 is found first in my path. I think this causes the issue I see when trying to run the scripts directly.  I assume others might experience this, so outside of uninstalling python2, adjusting my PATH order, or also using pip 2 to install confluent-kafka, is there a better remedy?

Reply:
see comment about virtualenv